src/index.tsx

```diff
@@ -85,7 +84,6 @@ const App: React.FC = () => (
           <DateProvider>
             <LocaleProvider>
               <MessageManagerProvider>
-                <ServiceWorker />
```

Comment:
**question:** isn't `ServiceWorker` needed for informing user about new version of dashboard?

Reply:
service worker was never been loaded, I want to create a new issue for that (enabling SW with vite)

Reply:
but yeah... you are right about that component 👍